app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
```

Comment:
We utilize [BEM class naming](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/). Given that `.pagination` is the block, other styles within this partial should be elements and modifiers. For example, it might make sense for this to be a modified version of the `pagination` class, which would dictate this class as `pagination--with-total`.

We also try and avoid unnecessary nesting and instead maintain a flat, low-specificity CSS architecture.